interfaces/builtin/modem_manager.go

```diff
@@ -47,6 +47,10 @@ var modemManagerPermanentSlotAppArmor = []byte(`
 # For ioctl TIOCSSERIAL ASYNC_CLOSING_WAIT_NONE
 capability sys_admin,
 
+# For {mbim,qmi}-proxy
+unix (create,bind,listen,send,receive) type=stream,
+/sys/devices/**/usb** r,
```

Comment:
What is this for? Should modem-manager instead use the 'raw-usb' interface which allows this access?